src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -1297,7 +1360,8 @@ public ModelBase GetModelInternal(Guid modelGuid)
         {
             ModelBase foundModel = (Connectors.FirstOrDefault(c => c.GUID == modelGuid)
                 ?? (ModelBase)Nodes.FirstOrDefault(node => node.GUID == modelGuid))
-                ?? Notes.FirstOrDefault(note => note.GUID == modelGuid);
+                ?? (Notes.FirstOrDefault(note => note.GUID == modelGuid)
+                ?? (ModelBase) Annotations.FirstOrDefault(annotation => annotation.GUID == modelGuid));
 
```

Comment:
This is also a fairly yucky looking.  We should use `as` rather than the cast which could throw an exception.